allennlp/data/dataset_readers/conll2003.py

```diff
@@ -66,13 +67,18 @@ class Conll2003DatasetReader(DatasetReader):
         Each will have its own namespace : `pos_tags`, `chunk_tags`, `ner_tags`.
         If you want to use one of the tags as a `feature` in your model, it should be
         specified here.
-    coding_scheme : `str`, optional (default=`IOB1`)
+    convert_to_coding_scheme : `str`, optional (default=`None`)
         Specifies the coding scheme for `ner_labels` and `chunk_labels`.
-        Valid options are `IOB1` and `BIOUL`.  The `IOB1` default maintains
+        Valid options is `BIOUL`.  If `None`, it default maintains
         the original IOB1 scheme in the CoNLL 2003 NER data.
         In the IOB1 scheme, I is a token inside a span, O is a token outside
         a span and B is the beginning of span immediately following another
         span of the same type.
+    coding_scheme : `str`, optional (default=`IOB1`)
+        This parameter is deprecated. If you specify `coding_scheme` to
+        `IOB1`, consider simply removing it or specifying `convert_to_coding_scheme`
+        to `None`. If you want to specify `BIOUL` for `coding_scheme`,
+        consider replacing it with `convert_to_coding_scheme`.
```

Comment:
✅

I think it would be good to mention above that the assumed coding scheme is `IOB1`. That wasn't clear to me when reading the original docstring.